packages/twenty-front/src/modules/action-menu/actions/record-actions/components/ManageFavoritesActionEffect.tsx

```diff
@@ -51,6 +51,7 @@ export const ManageFavoritesActionEffect = ({
 
     addActionMenuEntry({
       type: 'standard',
+      scope: 'record-selection',
```

Comment:
I feel this whole scope logic is duplicating the
```
const singleRecordActionEffects = [
  ManageFavoritesActionEffect,
  DeleteRecordsActionEffect,
];

const multipleRecordActionEffects = [
  ExportRecordsActionEffect,
  DeleteRecordsActionEffect,
];
```
logic

"no-record" (aka Global) vs "single-record" vs "multiple-records"

We should choose between the two ways of doing